install_alpine_dependencies.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -eu
+
+# basic build dependencies of various Django apps for Alpine 3.6
+apk -U add \
+  `# build-base metapackage install: binutils, fortify-headers, g++, gcc, libc-dev, make` \
+  build-base \
+  `# required to translate` \
+  gettext \
+  python2-dev \
+  `# shared dependencies of Pillow, pylibmc` \
+  zlib-dev \
+  `# Postgresql and psycopg2 dependencies` \
+  postgresql-dev \
+  `# Pillow dependencies` \
+  tiff-dev \
+  jpeg-dev \
+  freetype-dev \
+  lcms-dev \
+  libwebp-dev \
+  `# django-extensions` \
+  graphviz-dev \
+  `# hitch` \
+  py-setuptools \
+  python3-dev \
+  py-virtualenv \
+  py2-pip \
+  firefox-esr \
+  automake \
+  libtool \
+  readline \
+  readline-dev \
+  sqlite-dev \
+  libxml2 \
+  libxml2-dev \
+  libressl-dev \
+  bzip2-dev \
+  wget \
+  curl \
+  llvm4 \
+  `# federation` \
+  libxml2-dev \
+  libxslt-dev \
+  python3-dev \
+  `# socialhome` \
+  git \
+  linux-headers \
+  nodejs \
```

Comment:
Which Node is this btw? Currently I think anything goes (though I have only tested 4+), but to compile the Vue.js frontend bundle, Node 6 seems to be required. I have been meaning to add this already to the documentation, even though the Vue frontends are not ready yet.

Reply:
It's [this](https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.6) one.

Reply:
Thanks that is sufficient 👍